docs/_includes/admin-tasks-content.md

```diff
@@ -568,33 +625,31 @@ boxes. By default, users will then be shown a filtered list of categories releva
 
 #### Removing staff status from accounts
 
-To remove the staff status from an account visit the user page and
-uncheck the ‘staff’ checkbox. To prevent a user from logging in uncheck
-the ‘email verified’ checkbox.
+To remove the staff status from an account visit the 'Users' page, select the user in question and uncheck the ‘staff’ checkbox. 
+
+<img loading="lazy" alt="Staff tick box" src="/assets/img/ww-user-guide/WasteWorks%20staff%20user%20tick%20box.png" />
+
+To prevent a user from logging in uncheck the ‘email verified’ checkbox.
 
-You can do this in bulk on the Users screen by selecting the checkbox
-next to the user's name on the users page, selecting the ‘Remove staff
-permission’ checkbox and clicking submit.
+<img loading="lazy" alt="Email verified tick box" src="/assets/img/ww-user-guide/WasteWorks%20email%20verified%20tick%20box.png" />
 
-Note that if a user has access to the email address associated with an
-account they can re-enable their account, but will not have staff
-permissions.
+You can do this in bulk on the 'Users' screen by selecting the checkbox next to the user’s name on the 'Users' page, selecting the ‘Remove staff permission’ checkbox and clicking submit.
+
+<img loading="lazy" alt="Bulk user removal" src="/assets/img/ww-user-guide/WasteWorks%20bulk%20user%20removal.png" />
+
+**Please note:** If a user has access to the email address associated with an account they can re-enable their account, but will not have staff permissions.
 
 #### Removing accounts
 
-To remove an account, the Administrator-level member of staff should make contact with
-mySociety.
+To remove an account, the Administrator-level member of staff should make contact with SocietyWorks.
```

Comment:
Should they? They have access to the user anonymization button, so there's nothing stopping them 'removing' an account at the moment.